docs(common):update csp docs (#1284)
telerik/blazor-docs: troubleshooting/csp.md

      img-src 'self' data:" />
````

>tip If you do not use our CDN services, you can remove their domains. If you do not use the Templates of the Charts, you may also be able to remove `'unsafe-eval'`.

## Upcoming Enhancements

Some of the above-listed limitations will be addressed in a future version of Telerik UI for Blazor. The planned enhancements are:

> * Remove the necessity for specifying the `unsafe-eval` directive when the [Chart Templates]({%slug components/chart/label-template-format%}) are used.

Reviewer: ```suggestion
* Remove the need to specify `unsafe-eval` when [Chart Templates]({%slug components/chart/label-template-format%}) are used.
```

> * Remove the necessity for specifying the `unsafe-inline` directive.

Reviewer: ```suggestion
* Remove the need to specify `unsafe-inline`.
```

> * Fix the `font-src` rule by detaching font icons.

Author: ```suggestion
* Remove the need to specify `font-src` rule by detaching font icons.
```

> * Fix the `font-src` rule by detaching font icons.

Reviewer: What exactly do we fix?
